src/inspector/string_view.rs

```diff
@@ -146,6 +156,18 @@ where
 unsafe impl<'a, T> Send for CharacterArray<'a, T> where T: Copy {}
 unsafe impl<'a, T> Sync for CharacterArray<'a, T> where T: Sync {}
 
+impl fmt::Display for CharacterArray<'_, u8> {
+  fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+    f.write_str(&string::String::from_utf8_lossy(&*self))
```

Comment:
For debugging it's not such a big deal, but keep in mind that 8-bit V8 strings are Latin-8 and not UTF-8.
I would suggest:
```
write!(f, "{}", self.iter().map(|c| *c as char).collect::<String>()));
```